### PR TITLE
added specific input validation.

### PR DESCRIPTION
**Pull request by jesslane94**
- added regex expression so name only accepts alphabet A-Z/lower and uppercase.
- email should now allow other domains besides .com and .net
- phone number should be restricted to 0-9 only
- added these checks to server side as well.

### CODE REVIEW

**Comment by jesslane94** [comment]
i just realized i forgot to take out a console log statement. ill try to remember to remove that on my next branch I work on!

File: client/components/form.js [hunk 1]
```diff
@@ -58,6 +60,7 @@ export default function Form() {
     });
 
     if (error) {
+      console.log(error);
```
Comment (DalySoftware) [inline client/components/form.js:63]: Possibly console.warn() or console.error() would be more appropriate?

File: client/components/form.js [hunk 2]
```diff
@@ -14,17 +14,19 @@ import { Button } from "@mui/material";
 import { requestAppt } from "@/actions/form";
 
 // define schema
-
 const schema = Joi.object({
-  //! add regex to restrict to alpha only
-  name: Joi.string().alphanum().min(2).max(30).required(),
-  //! update tlds config (throws error if not present)
+  name: Joi.string()
+    .pattern(new RegExp(/^[A-Za-z]+$/))
```
Comment (DalySoftware) [inline client/components/form.js:19]: FYI this wouldn't be a great regex for names in the real world. Silly example: Björk tries to sign up for our app

**Review by DalySoftware — APPROVED** [review]

File: client/components/form.js (hunk 2 above)
Comment (jesslane94) [reply client/components/form.js:19]: Ahh that is a good point. I could look into how to deal with that if it's feasible haha

Comment (tdkent) [reply client/components/form.js:19]: Might be a good idea to update this at some point to include common symbols used in names like hyphens, commas, apostrophes, etc.

**Pull request merged**